roles/openshift_master/handlers/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-- name: restart openshift-master
-  service: name=openshift-master state=restarted
+- name: restart master
+  service: name={{ product_type }}-master state=restarted
```

Comment:
I'm guessing we should change the notification trigger to just `restart master` since that's the new name.